common/djangoapps/third_party_auth/tests/test_utils.py

```diff
@@ -38,3 +38,18 @@ def test_user_exists(self):
         self.assertTrue(
             user_exists({'username': 'TesT_User'})
         )
+
+    def test_convert_slug_provider_id(self):
+        """
+        Verify saml provider id/slug map to each other correctly.
+        """
+        provider_ids = {'saml-samltest': 'samltest', 'saml-example': 'example'}
+        for pid in provider_ids.keys():
```

Comment:
tiny nit: I don't think pid isn't a great name (pid to me means process identifier), but also it doesn't really describe what we're operating on. In the same way that I don't think in Java `Object o` is very useful. I would just call this `provider_id`

Reply:
ah I just changed this to key before seeing this (forgot the default value in a dictionary iterator is just the key, so switched to `for key in provider_ids`. But can also change it to provider_id